File: DuDoong-Common/src/main/java/band/gosrock/common/exception/BadLockIdentifierException.java
package band.gosrock.common.exception;

public class BadLockIdentifierException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new BadLockIdentifierException();

    private BadLockIdentifierException() {
        super(GlobalErrorCode.BAD_LOCK_IDENTIFIER);
    }
}

File: DuDoong-Common/src/main/kotlin/band/gosrock/common/exception/BadLockIdentifierException.kt
package band.gosrock.common.exception

class BadLockIdentifierException private constructor() : DuDoongCodeException(GlobalErrorCode.BAD_LOCK_IDENTIFIER) {
    companion object {
@JvmField
        val EXCEPTION: DuDoongCodeException = BadLockIdentifierException()
    }
}
